py3dtilers/Common/lod_tree.py

```diff
@@ -10,7 +11,7 @@ class LodTree(GeometryTree):
     The LodTree contains the root node(s) of the LOD hierarchy and the centroid of the whole tileset
     """
 
-    def __init__(self, groups: 'Groups', create_lod1=False, create_loa=False, with_texture=False, geometric_errors=[None, None, None]):
+    def __init__(self, groups: 'Groups', create_lod1=False, create_loa=False, with_texture=False, geometric_errors=[None, None, None], texture_lods=0):
```

Comment:
In the futur, we may have to think about how can we enhanced this constructor.
There is now a lot of parameter, maybe we can use an "option" object as holding all parameter ?